packages/shared/src/components/squads/SquadListingHeader.tsx

```diff
@@ -0,0 +1,41 @@
+import React, { ReactElement } from 'react';
+import { cloudinary } from '../../lib/image';
+import { Button } from '../buttons/Button';
+
+interface SquadListingHeaderProps {
+  isOwner?: boolean;
+}
+
+export const SquadListingHeader = ({
+  isOwner = false,
+}: SquadListingHeaderProps): ReactElement => {
+  return (
+    <div className="mb-4">
+      <div
+        className="flex flex-col items-center p-6 mb-1 text-center bg-center bg-cover rounded-24 bg-theme-bg-primary"
+        style={{
+          backgroundImage: `url(${cloudinary.squads.listingBanner})`,
+        }}
+      >
+        <div className="flex justify-center items-center mb-3 text-white typo-large-title">
+          Introducing Squads
+        </div>
+        <div className="mb-4 type-text3 text-theme-color-salt max-w-[40rem]">
+          Unleashing the magic of developer communities with Squads. An
+          opportunity to dive deep and go niche together with like-minded devs.
+        </div>
+        <Button
+          className="btn-primary"
+          onClick={() =>
+            isOwner ? alert('Join waitlist...') : alert('Create new squad...')
```

Comment:
Create new squad is not a new option we should already have this.
Also the waitlist will be a URL, I'll get these to you today.

Reply:
Join waitlist: `https://r.daily.dev/public-squad-waitlist` (please add to the URLS lib)